app/frontend/src/features/auth/email/ConfirmChangeEmail.stories.tsx

```diff
@@ -26,14 +25,14 @@ export default {
 
 interface ConfirmChangeEmailArgs {
   isLoading?: boolean;
-  shouldSucceed?: boolean;
+  resultState?: EmailConfirmationState | false;
```

Comment:
Not really sure how to do this

Reply:
Yeah, I see this is no longer the straightforward succeed or fail binary state since there are multiple states now. Maybe `null` better represents something has gone wrong (even though this is technically not an object, but with the type name it kinda still reads like one)?

Reply:
Yeah: I just wasn't sure how to use null :P